docs/en/latest/apisix-variable.md

```diff
@@ -49,5 +49,6 @@ additional variables.
 | service_name        | core       | Name of Service.                                                                    |                |
 | redis_cmd_line      | Redis      | The content of Redis command.                                                       |                |
 | rpc_time            | xRPC       | Time spent at the rpc request level.                                                |                |
+| resp_body           | core       | This variable is only for logger and requires the logger plugin to have a special configuration like `include_resp_body`. |                |
```

Comment:
We should explain what this variable is, and what decides its value

Reply:
done